Cargo.toml

```diff
@@ -18,15 +18,24 @@ __internal_disable_timer_in_integration_test = []
 [dependencies]
 libtock_core = { path = "core" }
 libtock_codegen = { path = "codegen" }
-futures = { version = "0.3.1", default-features = false, features = ["unstable", "cfg-target-has-atomic"] }
+futures = { version = "0.3.1", default-features = false, features = [
+    "unstable",
+    "cfg-target-has-atomic",
+] }
 
 [dev-dependencies]
 corepack = { version = "0.4.0", default-features = false, features = ["alloc"] }
 # We pin the serde version because newer serde versions may not be compatible
 # with the nightly toolchain used by libtock-rs.
-serde = { version = "=1.0.114", default-features = false, features = ["derive"] }
+serde = { version = "=1.0.114", default-features = false, features = [
+    "derive",
+] }
 ctap2-authenticator = { git = "https://gitlab.com/ctap2-authenticator/ctap2-authenticator.git" }
-p256 = { version = "0.7" , default-features = false, features = ["arithmetic", "ecdsa", "ecdsa-core"] }
+p256 = { version = "0.7", default-features = false, features = [
+    "arithmetic",
+    "ecdsa",
+    "ecdsa-core",
+] }
```

Comment:
Most of these changes seem to be from an autoformatter. Is that the case?

Reply:
Yes. I'm not sure what the standards of this project are, but I figured auto-formatting would be acceptable.

Reply:
I'm not a fan of ad-hoc autoformatting because then you end up with changes unrelated to the purpose of the PR that contains them (i.e. most of these changes are not related to low level debug), but adding a `.toml` formatting check to `make test` seems like a good idea.

What autoformatter did you use?

Reply:
Just `cargo fmt`. Do you want me to remove these auto-formatting changes from this PR?

Reply:
Ah scratch that, I believe it was actually VSCode. I don't know the formatter it uses.

Reply:
Yeah, I would prefer to have the formatting changes removed. The problem with VSCode's formatting is it's difficult for contributors using other editors to reproduce, and difficult to check in CI. How difficult is it to turn off VSCode's autoformatting of TOML files?

Looking at https://github.com/toml-lang/toml/issues/532, it looks like all the TOML autoformatters are either one-man projects (which we don't necessarily want to depend on), or are built in to editors (which have the same issues as VSCode's formatting).

Reply:
OK, I finally tracked it down, it was actually this extension specifically: https://marketplace.visualstudio.com/items?itemName=tamasfe.even-better-toml. I disabled auto-formatting for toml files with this:

```json
"[toml]": {
    "editor.formatOnSave": false
}
```

And then pushed a new commit.